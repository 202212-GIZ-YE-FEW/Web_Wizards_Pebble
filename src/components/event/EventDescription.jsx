import Title from "./Title";

const EventDescription = (props) => {
    const eventData = props.eventData;

    return (
        <div className='order-1 lg:w-7/12 w-full px-2'>
            <Title text='Eevent Description:' />
            <p className='  lg:text-lg  md:text-md'>{eventData.description}</p>
        </div>
    );
};

export default EventDescription;
