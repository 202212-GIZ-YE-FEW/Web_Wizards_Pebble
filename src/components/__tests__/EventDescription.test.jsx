import renderer from "react-test-renderer";

import EventDescription from "../event/EventDescription";

it("renders correctly", () => {
    const eventData = {
        title: "Meeting Brainstorm Energy CIeccan lzmir",
        location: "Izmir, Turkey",
        date: "ahhallsei 26 Augest Kapisi",
        organizer: "Sufyan",
        attendees: ["Sufyan", "Adeeb", "Abdullah", "Azzam", "Hamad"],
    };
    const tree = renderer
        .create(<EventDescription eventData={eventData} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
