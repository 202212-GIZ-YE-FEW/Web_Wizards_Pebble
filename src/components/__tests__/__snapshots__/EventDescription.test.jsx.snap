// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="order-1 lg:w-7/12 w-full px-2"
>
  <h2
    className="text-2xl font-semibold mb-2 "
  >
    Eevent Description:
  </h2>
  <p
    className="  lg:text-lg  md:text-md"
  />
</div>
`;
